libraries/chain/resource_limits.cpp

```diff
@@ -412,5 +412,16 @@ account_resource_limit resource_limits_manager::get_account_net_limit_ex( const
    return arl;
 }
 
+bool resource_limits_manager::is_greylisted(const account_name &name) const {
```

Comment:
Our other black/whitelists are managed and maintained in the controller, this should probably go in the same place. 

Each `resource_limits_manager` method can take a single bool that indicates whether the operation in question should allow access or deny access to the elastic limits as opposed to now where it takes a flag which indicates the greylist may apply and then checks the greylist.   That will lower the burden for future devs who will find all of the black/grey/whitelists in the same system and reduce the amount of "application" logic in the `resouce_limits_manager`